app/src/main/java/com/example/footballmanager/data/AppModule.kt

```diff
@@ -35,4 +41,14 @@ object FootballApi {
     @Singleton
     fun provideApiService(retrofit: Retrofit): FootballApiService =
         retrofit.create(FootballApiService::class.java)
+
+    @Provides
+    @Singleton
+    fun provideDatabase(app: Application) =
+        Room.databaseBuilder(app, MatchesRoomDatabase::class.java, DATABASE_NAME)
+            .fallbackToDestructiveMigration()
```

Comment:
btw you can familiarize yourself with the migrations in Room 
https://developer.android.com/training/data-storage/room/migrating-db-versions

Reply:
Done!